app/Model/Identifier.php

```diff
@@ -129,7 +129,7 @@ class Identifier extends AppModel {
                                                  IdentifierEnum::StudentSOR,
                                                  IdentifierEnum::UID))),
         'required' => true,
-        'allowEmpty' => false
+        'allowEmpty' => true
       )
```

Comment:
Is this right? Why would we make this field required but then allow it to be empty?

Reply:
Yes this is intentional. We need to be able to save an empty value.
```php
$identifier = '';
```

Reply:
@boshrin 
i reverted the change.